bip-0078.mediawiki

```diff
@@ -187,10 +187,10 @@ The well-known error codes are:
 |The receiver rejected the original PSBT.
 |}
 
-The receiver is allowed to return implementation specific errors which may assist the sender to diagnose any issue.
+The receiver is allowed to return implementation-specific errors which may assist the sender to diagnose any issue.
 
 However, it is important that error codes that are not well-known and that the message do not appear on the sender's software user interface.
```

Comment:
I ended up leaving this sentence alone. I can open up another PR if the BIP author chimes in on discussion or if there is  agreement on how to reword it without changing the meaning.